pytest_plugins/metadata_markers.py

```diff
@@ -24,8 +24,8 @@ def pytest_addoption(parser):
         help='Comma separated list of component names to include in test collection',
     )
     parser.addoption(
-        '--assignee',
-        help='Comma separated list of assignees to include in test collection',
+        '--team',
+        help='Comma separated list of teams to include in test collection',
```

Comment:
Can we not use the old Assignee field for assigning test modules to teams instead of creating a new field? Just wondering what the rationale is here, non-blocking

Reply:
We could have but their is no point in keeping `assignee` field that should point to person but we want to point it to `team`.

Anyways there is just a difference of `key` name here, the functionality  should remain same.